### ISSUE TITLE
Asbru always saves session log as file

### ISSUE THREAD

**Issue by ValdikSS** [asbru-cm/asbru-cm#1112]
**Describe the bug**

Even when "save session log" function is disabled, asbru records all the output of the session and saves it into `$CFG_DIR/tmp/asbru_PID{PID}_nX.txt` file.
The file is deleted after the session is closed, but during that time any software with access to `~/.config/asbru` can read the contents of any session. The file is created with default umask:

`-rw-r--r--. 1 valdikss valdikss   753 сен 16 16:16 asbru_PID{14158}_n2.txt`

Any user could read it. Given that many distros still have `755` on `/home`, that means that any software can read session contents live.

This also causes unnecessary disk wear.

**To Reproduce**
1. Run Asbru
2. Connect to any SSH server
3. Monitor contents of `~/.config/asbru/tmp` directory

**Expected behavior**

No session log is saved as a file

**Environment (please complete the following information):**
 - OS: Fedora 39
 - Ásbrú Version: 6.4.0

**Comment by alexus-m** [comment]
Hello!

May be the current behavior (i.e. always to store logs in a temporary files) could be acceptable in a debug mode.

**Comment by Scud0** [comment]
Seeing the same behaviour here on Mint with Asbru v6.4.0. 

In my case the log files filled up the entire disk, that ran out of space. Which causes all kinds of problems on it own.

**Comment by gfrenoy** [comment]
Thanks for the heads-up.  I never really had any issue with this but I agree this is not the behaviour by default that would be expected.

The session logs should only be created when explicitely asked ; either globally, either for a particular session.

I can replace line 164 of `asbru-cm/lib/PACTerminal.pm` with:
```perl
$self->{_LOGFILE} = '/dev/null';
```

Would that make sense ?

**Comment by ValdikSS** [comment]
> Would that make sense ?

That's exactly what I did and been running since creating this issue.

**Comment by gfrenoy** [comment]
> That's exactly what I did and been running since creating this issue.

Very good, thanks for the confirmation.  Let me submit a change for that then.

**Comment by KlaasT** [comment]
@gfrenoy Do you know if this issue can be closed? Looks fixed to me, when I read through the commits.

**Comment by ValdikSS** [comment]
@KlaasT it's only in git master, not it any release yet, maybe that's why.
Or should I close the ticket?

**Comment by gfrenoy** [comment]
Yes, this is fixed in the latest 6.4.1 version and also available in loki.    Let's close the issue.
